packages/dids/package.json

```diff
@@ -101,7 +100,6 @@
     "@web/test-runner-playwright": "0.11.0",
     "c8": "9.0.0",
     "chai": "4.3.10",
-    "chai-as-promised": "7.1.1",
```

Comment:
I encourage moving off of this package in the long term because it breaks compat with codemods and the larger js echosystem like Jest, Vite or Bun's test runner

Reply:
💯 Have been slowing trying to remove it to make a migration to a different test framework easier.